identity/identity.proto

```diff
@@ -200,6 +200,27 @@ message Capability {
     Type type = 1;
   }
 
+  // EncryptionKeyRotation contains the features of the EncryptionKeyRotation
+  // operation that the CSI-driver supports.
+  message EncryptionKeyRotation {
```

Comment:
The plan is to set the request to all the nodes wherever the app is running. and is it up to the csi driver how it wants to rotate the keys?

Reply:
The rpc is sent only to one of the nodes in case of RWX. Upon receiving the request it is upto the driver to handle the rotation.

Reply:
This spec is unrelated to the implementation in the sense that we want to have this service advertised on all the nodes, irrespective of the volume types.

Reply:
The implementation should follow the rules defined in the specification repository. We should not make assumptions that are not specified in the rules, and we should only make RPC calls to specific nodes. If we are advertising that RPC from all node plugins, according to the specification, we need to send RPC calls to all node plugins where the app pod is mounted. It will be up to the node plugin to decide how to handle the RPC call. In the spec and kubernetes-csi-addons, we should not make assumptions and send requests to specific nodes unless it is predefined in the spec or the CSI driver explicitly states support for reclaiming space for RWX on only one node.

Reply:
Resolving this as discussed.